Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -98,6 +98,22 @@ public extension Node where Context: HTMLNamableContext {
     }
 }
 
+public extension Attribute where Context: HTMLNamableContext {
```

Comment:
Does it make sense to constrain this extension to `Context == HTML.MetaContext` instead? Given that not all namable items can have a property assigned to them, right? Or perhaps I'm missing something?